backend/src/main/java/com/github/mysterix5/vover/security/SecurityConfig.java

```diff
@@ -28,6 +28,7 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
         http.csrf().disable().authorizeRequests()
                 .antMatchers(HttpMethod.POST, "/api/auth/login").permitAll()
                 .antMatchers(HttpMethod.POST, "/api/auth/register").permitAll()
+                .antMatchers("/api/admin/**").hasAnyRole("admin")
```

Comment:
```suggestion
                .antMatchers("/api/admin/**").hasRole("admin")
```
Since this path is only open for one role